platform/jewel/ui/src/main/kotlin/org/jetbrains/jewel/ui/component/PlatformIcon.kt

```diff
@@ -32,7 +32,7 @@ public fun PlatformIcon(
 @Composable
 public fun PlatformIcon(
     key: IntelliJIconKey,
-    contentDescription: String?,
+    contentDescription: String,
```

Comment:
Why this API change? This is strictly wrong. `Icon` and `Image` take a nullable content description because it might not need to be announced, if it has no reason to be. Often, it doesn't, either because the icon itself is only decorative, or because while it is used in an actionable component, the parent component is the one that actually needs to announce itself, not its icon.